nameko_grpc/connection.py

```diff
@@ -199,6 +238,7 @@ def window_updated(self, event):
         Any data waiting to be sent on the stream may fit in the window now.
         """
         log.debug("window updated, stream %s", event.stream_id)
+        self.send_headers(event.stream_id)
```

Comment:
Need to send headers before data if they exist.

Reply:
Was this a pre-existing bug that you've happened to identify as part of this work? If so, how did you identity it?

Reply:
arguably an existing bug that came up through existing tests after changing the stop method to shutdown with a goway frame first.